src/main/java/com/box/sdk/BoxRetentionPolicy.java

```diff
@@ -201,7 +202,7 @@ public BoxRetentionPolicy(BoxAPIConnection api, String id) {
         }
         request.setBody(requestJSON.toString());
         BoxJSONResponse response = (BoxJSONResponse) request.send();
-        JsonObject responseJSON = JsonObject.readFrom(response.getJSON());
+        JsonObject responseJSON = Json.parse(response.getJSON()).asObject();

```

Comment:
I guess it is only a syntax change? Applies to other files with same change.

Reply:
JsonObject.readFrom is deprecated. We should be using  Json.parse